make_and_run_bmi_pass_forcings.sh

```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
-touch run_cfe_pass_forcings
-mv -f run_cfe_pass_forcings z_trash
-gcc -lm ./src/main_pass_forcings.c ./src/cfe.c ./src/bmi_cfe.c ../AORC/src/aorc.c ../AORC/src/bmi_aorc.c  -o run_cfe_bmi_pass_forcings
-./run_cfe_bmi_pass_forcings ./configs/cat_89_bmi_config_cfe_pass.txt ./configs/cat_89_bmi_config_aorc.txt
+gcc -lm ./src/main_pass_forcings.c ./src/cfe.c ./src/bmi_cfe.c ../alt-modular/Modules/AORC/src/aorc.c ../alt-modular/Modules/AORC/src/bmi_aorc.c  -o run_cfe_bmi_pass_forcings
```

Comment:
`../*/AORC/*` path no longer exists.  Either copy it directly into this repo or include it as a [git submodule](https://git-scm.com/book/en/v2/Git-Tools-Submodules)

Reply:
If just for testing, simply remove from repo and add to .gitignore

Reply:
Yeah, I guess we'll have to do that. I thought we would want to include test runs in the repos, either for contributing to development here (like new CFE functionality) or as an example for doing BMI for their own models. But there are other resources people can look at for examples, so we don't need to host them.